.gitlab-ci.yml

```diff
@@ -34,13 +36,20 @@ stages:
   - infra-prod
   - production
 
+default:
+  cache:
+    paths:
+      # $CI_PROJECT_DIR is /builds/senbox-org/snap-engine/ on ci
+      - "${CI_PROJECT_DIR}/testData"
+
 mvn-build:
+  before_script:
```

Comment:
This before_script will override the one comming from `include` scripts and pipeline will fail